### ISSUE TITLE
I use from progress.bar Import bar as bar. The program will run normally, but no progress bar will be output

### ISSUE THREAD

**Issue by Wan-Dou** [YU1ut/MixMatch-pytorch#36]
I use from progress.bar Import bar as bar. The program will run normally, but no progress bar will be output

**Comment by Wan-Dou** [comment]
from utils import Bar, Logger, AverageMeter, accuracy, mkdir_p, savefig，I can‘t find the Bar in utils

**Comment by shenlanvs001** [comment]
I face the same question : the loss of train becomes smaller , but the loss of val becomes larger.
My own dataset has 200 classes

**Comment by Lotk1103** [comment]
> I face the same question : the loss of train becomes smaller , but the loss of val becomes larger. My own dataset has 200 classes

Hey, do you solve this problem? I meet the same question, is it realted to dataset imbalance?

**Comment by SuperJunier666** [comment]
我也遇见了这个问题，我是用这个方法做分割
